CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+
+### Running tests
+
+For running the tests you require only need start rabbitmq in the background.
+The profile that is created during the tests will automatically check its configuration.
+To run all tests use
+
+```console
+pytest
+```
+
+To run only backend tests run
+
+```console
+pytest -m backend
+```
+
+To run only frontend tests
+```console
+pytest -m frontend
+```
+
+To not run these tests you can use the markers in the following way
+
+```console
+pytest -m "not backend and not frontend"
+```
+
+To debug the frontend tests you often want to see what happens in the tests.
+By default they are run in headless mode, so no browser is shown.
+To run the frontend tests in headed mode for you have to set an environment variable like this
+```console
+PYTEST_PLAYWRIGHT_HEADLESS=no pytest -m frontend
```

Comment:
The test seems blocked in the second test (test_workgraph) if I run this headed mode.

Reply:
Ah I see it is because the web server is already started in the CI and this blocks the start of a new one.
https://github.com/aiidateam/aiida-workgraph/blob/76047ee4c77b594193c0891dc4fd68c3012bf1f8/.github/workflows/ci.yaml#L96 
I will remove it to see if the CI works, but in any case I need to add proper error handling from the tests.

Reply:
Added now a check for the port in tests (see most recent commit).